fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBRecordStore.java

```diff
@@ -2431,11 +2431,11 @@ private boolean areAllRecordTypesSince(@Nullable Collection<RecordType> recordTy
         final boolean newStore = reason == RebuildIndexReason.NEW_STORE;
         if (newStore ? LOGGER.isDebugEnabled() : LOGGER.isInfoEnabled()) {
             final KeyValueLogMessage msg = KeyValueLogMessage.build("rebuilding index with no record",
-                    LogMessageKeys.INDEX_NAME, index.getName(),
-                    "indexVersion", index.getLastModifiedVersion(),
-                    "reason", reason.name(),
-                    subspaceProvider.logKey(), subspaceProvider.toString(context),
-                    LogMessageKeys.SUBSPACE_KEY, index.getSubspaceKey());
+                                            LogMessageKeys.INDEX_NAME    , index.getName(),
+                                            LogMessageKeys.INDEX_VERSION , index.getLastModifiedVersion(),
+                                            LogMessageKeys.REASON        , reason.name(),
+                                            subspaceProvider.logKey()    , subspaceProvider.toString(context),
+                                            LogMessageKeys.SUBSPACE_KEY  , index.getSubspaceKey());
```

Comment:
Remove column alignment.  I'll stop adding more such comments from here on.. :)

Reply:
DONE
- Removed column alignment everywhere it was added in previous commit.